observability/otel/otel-forwarding-logs.adoc

```diff
@@ -0,0 +1,202 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-forwarding-logs"]
+= Forwarding Logs to a LokiStack instance
+include::_attributes/common-attributes.adoc[]
+:context: otel-forwarding-logs
+
+You can deploy and configure the OpenTelemetry Collector to forward logs to a LokiStack instance. You can deploy the OpenTelemetry Collector in the deployment mode by using the specified processors, receivers, and exporters.
```

Comment:
```suggestion
You can deploy and configure the OpenTelemetry Collector to forward logs to a LokiStack instance. You can configure OpenTelemetry Collector for deployment mode by using the specified processors, receivers, and exporters.
```
or something; IDK "deploy in deployment mode" doesn't seem to mean much

Reply:
Thank you. I have consulted with the developer and then rewrote this sentence.